resources/prosody-plugins/mod_conference_duration_component.lua

```diff
@@ -0,0 +1,66 @@
+local st = require "util.stanza";
+local socket = require "socket";
+local json = require "util.json";
+local ext_events = module:require "ext_events";
+local it = require "util.iterators";
+
+-- we use async to detect Prosody 0.10 and earlier
+local have_async = pcall(require, "util.async");
+if not have_async then
+    module:log("warn", "speaker stats will not work with Prosody version 0.10 or less.");
```

Comment:
Maybe just fix the message to not say 'speaker stats' :)

Reply:
Fixed.